### ISSUE TITLE
Remove this

### ISSUE THREAD

**Issue by hashaam-zahid** [BogdanOtava/CS50x#1]
I am writing to bring to your attention a repository on GitHub CS50 that appears to be in violation of the academic policy of Harvard University. The repository in question contains solutions to assignments from the CS50x course, offered by Harvard's Extension School. Remove or make it private 
Looking forward

**Comment by eswarpilla** [comment]
hlo week 9 finance
you told to set api key
but how to get an api key and how to set an api key

**Comment by SophisticatedStupid** [comment]
> I am writing to bring to your attention a repository on GitHub CS50 that appears to be in violation of the academic policy of Harvard University. The repository in question contains solutions to assignments from the CS50x course, offered by Harvard's Extension School. Remove or make it private Looking forward

you are wrong bozo, it will be the problem of cheaters, not his. He can show and inspire other coders to think differently
